plutus-core/plutus-core/src/PlutusCore.hs

```diff
@@ -71,10 +71,8 @@ module PlutusCore
     -- * DeBruijn representation
     , DeBruijn (..)
     , NamedDeBruijn (..)
-    , deBruijnProgram
-    , deBruijnTerm
-    , unDeBruijnProgram
-    , unDeBruijnTerm
+    , runDeBruijnT
+    ,namedTyDeBruijnM,namedTermDeBruijnM,unNamedTyDeBruijnM,unNamedTermDeBruijnM
```

Comment:
tried to use both the old (to/from NamedDeBruijn) and the new (to/from DeBruijn) and came up with these ugly names.